smriprep/workflows/anatomical.py

```diff
@@ -529,7 +530,9 @@ def _check_img(img):
         name="surface_recon_wf", omp_nthreads=omp_nthreads, hires=hires
     )
     applyrefined = pe.Node(fsl.ApplyMask(), name="applyrefined")
-    sphere_reg_wf = init_sphere_reg_wf(name="sphere_reg_wf")
+    sphere_reg_wf = init_sphere_reg_wf(msm_sulc=msm_sulc, name="sphere_reg_wf")
+    if msm_sulc:
+        workflow.connect(surface_recon_wf, 'outputnode.sulc', sphere_reg_wf, 'inputnode.sulc')
```

Comment:
Since it's just connecting input/output nodes, this seems safe to do unconditionally.

Unless you're just wanting to clearly mark "this connection is for MSM only"?